### ISSUE TITLE
unexpected trailing "/" issue when using "python -m SimpleHTTPServer"

### ISSUE THREAD

**Issue by stephenguerin** [backspaces/asx#11]
When I successful run  http://backspaces.net/temp/asx/models?fire, chrome (or hostgo's webserver) changes the url to:
  http://backspaces.net/temp/asx/models/?fire

note the inserted slash between "models" and "?fire"

I downloaded http://backspaces.net/temp/asx.zip, unzipped and am running "python -m SimpleHTTPServer" with "asx" folder as my root directory. Running
   http://localhost:8000/models?fire

The URL gets converted to:
  http://localhost:8000/models?fire/

note trailing slash inserted at end instead of between "models" and "?fire"

This screws up the queryString parsing in index.html 

BTW running http://localhost:8000/models/ resolves fine to the default diffuse.js model. In the console I get:
  running: ./scripts/diffuse.js dir: scripts name: diffuse.js

Also, running from file:///{myLocalAsxDirectory}/models/index.html?fire resolves fine.

Here's a Python issue about the trailing "/" behavior:
  https://bugs.python.org/issue23112

Can you reproduce this with "python -m SimpleHTTPServer" on your machine. It could be a python version issue and it may now be resolved. Though we may want to change index.html to gracefully handle this on older versions of Python.

**Comment by backspaces** [comment]
> http://backspaces.net/temp/asx/models/?fire
> note the inserted slash between "models" and "?fire"

I think this is standard behavior, I noticed it too. But it results in the same behavior: running the index.html file correctly

> The URL gets converted to: http://localhost:8000/models?fire/
> note trailing slash inserted at end instead of between "models" and "?fire"

This appears to be caused by the Python issue you refer to. And the last comment there was in 2014 so my guess is they do not plan to fix it! 

My bet is that http://localhost:8000/models?scripts/fire just might work correctly. Recall we have a dual build, es6 modules as well as vanilla iife scripts, with the latter the default for now (but hopefully not for long)

Thank you for trying it on a variety of node/apache installs successfully too (as we discussed off-list), leads me to presume it is not worth fixing.
